subgraphs/abracadabra/src/common/setters.ts

```diff
@@ -0,0 +1,88 @@
+import { Address, BigInt } from "@graphprotocol/graph-ts";
+import { Market, _LiquidationCache } from "../../generated/schema";
+import { cauldron } from "../../generated/templates/cauldron/cauldron";
+import { getOrCreateLendingProtocol, getOrCreateToken } from "./getters";
+import { BIGDECIMAL_ZERO, BIGDECIMAL_ONE, BIGINT_ZERO, SECONDS_PER_YEAR, MIM } from "../common/constants";
+import { bigIntToBigDecimal } from "./utils/numbers";
+import { LogRemoveCollateral } from "../../generated/templates/cauldron/cauldron";
+
+export function updateProtocolMarketList(marketAddress: string): void {
+  let protocol = getOrCreateLendingProtocol();
+  let marketIdList = protocol.marketIdList;
+  marketIdList.push(marketAddress);
+  protocol.marketIdList = marketIdList;
+  protocol.save();
+}
+
+export function createMarket(marketAddress: string, blockNumber: BigInt, blockTimestamp: BigInt): void {
+  let MarketEntity = new Market(marketAddress);
+  let MarketContract = cauldron.bind(Address.fromString(marketAddress));
+  let collateralCall = MarketContract.try_collateral();
+  if (!collateralCall.reverted) {
+    let inputToken = getOrCreateToken(collateralCall.value);
+    MarketEntity.protocol = getOrCreateLendingProtocol().id;
+    MarketEntity.inputTokens = [inputToken.id];
+    MarketEntity.outputToken = MIM;
+    MarketEntity.totalValueLockedUSD = BIGDECIMAL_ZERO;
+    MarketEntity.totalVolumeUSD = BIGDECIMAL_ZERO;
+    MarketEntity.inputTokenBalances = [BIGINT_ZERO];
+    MarketEntity.outputToken = getOrCreateToken(Address.fromString(MIM)).id;
+    MarketEntity.outputTokenSupply = BIGINT_ZERO;
+    MarketEntity.outputTokenPriceUSD = BIGDECIMAL_ZERO;
+    MarketEntity.createdTimestamp = blockTimestamp;
+    MarketEntity.createdBlockNumber = blockNumber;
+    MarketEntity.name = inputToken.name + " Market";
+    MarketEntity.isActive = true;
+    MarketEntity.canUseAsCollateral = true;
+    MarketEntity.canBorrowFrom = true;
+
+    if (marketAddress.toLowerCase() == "0x551a7cff4de931f32893c928bbc3d25bf1fc5147".toLowerCase()) {
```

Comment:
Can you change all the addresses into named constants (e.g. `0x551a7cff4de931f32893c928bbc3d25bf1fc5147`)